data/meson.build

```
@@ -11,17 +11,24 @@ foreach i : icon_sizes
     )
 endforeach
 
-install_data(
-    'com.github.naaando.rush.gschema.xml',
-    install_dir: join_paths(get_option('datadir'), 'glib-2.0', 'schemas')
+i18n.merge_file(
+    input: meson.project_name() + '.desktop.in',
+    output: meson.project_name() + '.desktop',
+    po_dir: join_paths(meson.source_root(), 'po', 'extra'),
+    type: 'desktop',
+    install: true,
+    install_dir: join_paths(get_option('datadir'), 'applications')
 )
 
-install_data (
-    meson.project_name () + '.desktop',
-    install_dir: join_paths (get_option ('datadir'), 'applications')
+i18n.merge_file(
+    input: meson.project_name() + '.appdata.xml.in',
+    output: meson.project_name() + '.appdata.xml',
+    po_dir: join_paths(meson.source_root(), 'po', 'extra'),
+    install: true,
+    install_dir: join_paths(get_option('datadir'), 'metainfo')
 )
 
-install_data (
-    meson.project_name () + '.appdata.xml',
-    install_dir: join_paths (get_option ('datadir'), 'metainfo')
-)
+install_data(
+    'com.github.naaando.rush.gschema.xml',
```

```suggestion
    meson.project_name () + '.gschema.xml',
```

The current notation also works though.